Exiled.Events/EventArgs/Player/ActivatingGeneratorEventArgs.cs

```diff
@@ -48,7 +48,7 @@ public ActivatingGeneratorEventArgs(Player player, Scp079Generator generator, bo
         public Generator Generator { get; }
 
         /// <summary>
-        ///     Gets the player who's filpping the switch of the generator.
+        ///     Gets the player who's filpping the switch to the generator.
```

Comment:
Same as above

Reply:
This change is grammatically more correct, the switch is possive to the generator, so switch to a generator is more accurate. You wouldn't say "I flipped the switch of the generator" you would say "I flipped a switch to a generator"